src/dmd/root/filename.d

```diff
@@ -631,71 +631,74 @@ nothrow:
         }
     }
 
+    /**
+       Ensure that the provided path exists
+
+       Accepts a path to either a file or a directory.
+       In the former case, the `basepath` (path to the containing directory)
+       will be checked for existence, and created if it does not exists.
+       In the later case, the directory pointed to will be checked for existence
+       and created if needed.
+
+       Params:
+         path = a path to a file or a directory
+
```

Comment:
All these blank lines detract a bit from readability, as one has less code that fits on the screen.

Reply:
Please keep the newlines, it's consistent with coding conventions used in other dlang repositories, and it prevents the documentation from becoming a blob of text that is difficult to visually parse and glance through.